soc/raspberrypi/rp2xxx/Kconfig

```diff
@@ -14,10 +15,32 @@ config SOC_SERIES_RP2XXX
 	select XIP
 	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
 	help
-	  Enable support for Raspberry Pi RP2 MCU series
+	  Enable support for Raspberry Pi RP20 MCU series
+
+config SOC_SERIES_RP23XX
+	select ARM
+	select CPU_CORTEX_M33
+	select CPU_CORTEX_M_HAS_SYSTICK
+	select CPU_CORTEX_M_HAS_VTOR
+	select CPU_HAS_ARM_MPU
+	select HAS_RPI_PICO
+	select XIP
+	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
+	select PLATFORM_SPECIFIC_INIT
+	help
+		Enable support for Raspberry Pi RP23 MCU series
```

Comment:
1x tab followed by 2x spaces for help text indent

Reply:
Thanks, fixed.